src/scripts/main.js

```diff
@@ -1,3 +1,304 @@
 'use strict';
 
-// write your code here
+let gameField = [
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+  [0, 0, 0, 0],
+];
+
+let score = 0;
+const cellsLength = 4;
+let gameClickBehavior = false;
+
+const button = document.querySelector('.button');
+const fieldRow = document.querySelectorAll('.field-row');
+const gameScore = document.querySelector('.game-score');
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+button.addEventListener('click', () => {
+  gameClickBehavior = true;
+
+  if (button.classList.contains('start')) {
+    button.classList.remove('start');
+    button.classList.add('restart');
+    button.innerHTML = 'Restart';
+    messageStart.classList.add('hidden');
+
+    handleAddNumber();
+    handleAddNumber();
+    updateFieldCell();
+  } else if (button.classList.contains('restart')) {
```

Comment:
What do you think about immediately start the game after press on _restart_ (now you should click on _start_ after _restart_)?